app/assets/stylesheets/my_modules/protocols/index.scss

```diff
@@ -680,3 +682,52 @@
   display: inline-block;
   margin-left: 4px;
 }
+
+.confirm-link-update-modal {
```

Comment:
Do not invent a new modal, just use the current. There is just an error In mockup.

Reply:
The new Figma style is different for protocol repository revamp